vagrant_support/checksetup.yml

```diff
@@ -18,10 +20,11 @@
     owner: vagrant
     group: vagrant
     mode: 0644
+  when: LAZY == 0
 
 - name: run checksetup
   become: false
-  shell: sg apache -c '/usr/local/bin/bmo-checksetup'
+  shell: sg apache -c '/usr/local/bin/bmo-checksetup --no-templates'
```

Comment:
This won't work, although maybe it should.  You need to look at vagrant_support/bmo-checksetup.j2